.github/workflows/build.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         run: |
           sudo apt-get install -y musl-tools --no-install-recommends
       - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
-      - uses: dtolnay/rust-toolchain@d76c2a93c593483c9892fd8d2538734f73ed8588 #stable
+      - uses: dtolnay/rust-toolchain@439cf607258077187679211f12aa6f19af4a0af7 #stable
```

Comment:
https://github.com/dtolnay/rust-toolchain/commit/439cf607258077187679211f12aa6f19af4a0af7 is master